arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -2536,27 +2502,23 @@
             "Unknown"
           ],
           "x-ms-enum": {
-            "name": "EntityStatus",
+            "name": "entityStatus",
```

Comment:
same as above. Define it in one place with correct casing and reference it wherever required.